server/src/review/review.data-access.ts

```diff
@@ -49,6 +49,11 @@ export const findClassReviews = async (courseId: string) => {
 export const insertReview = async (review: Review) => {
   const newReview = new Reviews(review);
   await newReview.save();
+  const courseId = review['class'];
+  await Classes.updateOne(
+    { _id: courseId },
+    { $inc: { freshness: 1 } }
```

Comment:
A couple thoughts on updating freshness: first, I believe the scale that you and Will were discussing was out of 100. Considering that we don't get a ton of reviews, we could consider a couple different ideas to better work within the site:

- Increase freshness by more (i.e. 5) each review
- Perhaps by a constant factor (increase by 10%) of the current freshness. This might represent that a lot of ongoing activity exponentially increases the freshness (so it increases quickly?), but could also foresee issues with this and the freshness limit

Those would probably depend on how quickly freshness depreciates. We'd also want to have a cap on freshness so we cannot increment the field past its max limit.

Reply:
Possible options of the top of my head:

1. Current implementation (increment freshness by one per new review) 
-> when we run updateSummaries in a new semester, we have to pull freshness and divide by number of reviews. => have some threshold here (ie.

```python
if freshness/number of reviews >= 30%: 
   resummarize()
else: 
   continue
```


2. We pre-calculate freshness upon insert